### ISSUE TITLE
Tracking Issue: Condense effects

### ISSUE THREAD

**Issue by TikaelSol** [foundryvtt/pf2e#4144]
Here's a list of spells that should be condensed to a single effect then migrated:

- [ ] Bestial Curse
- [ ] Blessing of Defiance
- [ ] Clawsong
- [x] Community Repair
- [ ] Corrosive Body
- [ ] Discern Secrets
- [ ] Divine Vessel
- [ ] Dragon Wings
- [ ] Draw Ire
- [ ] Dread Ambience
- [ ] Elemental Gift
- [ ] Elemental Motion (Will need some sorcerer bloodline reworking)
- [x] Enlarge
- [ ] Evolution Surge
- [ ] Fungal Infestation
- [ ] Impeccable Flow
- [x] Lay on Hands
- [ ] Primal Summons
- [ ] Rapid Adaptation
- [ ] Shifting Form
- [ ] Swampcall
- [ ] Sweet Dream
- [ ] Traveler's Transit
- [ ] Warding Aggression
- [ ] Weapon Surge

Feat / Feature effects

- [ ] Liberating Step (oaths)
- [x] Overdrive
- [ ] Titan's Stature (merge into Giant's Stature)

Equipment effects

- [ ] Ring of torag (changed to toggle)

Bestiary effects

- [ ] Energy Ward

**Comment by dogstarrb** [comment]
M-Z candidates:

Primal Summons
Rapid Adaptation
Shifting Form
Swampcall
Sweet Dream
Traveler's Transit
Warding Aggression
Weapon Surge

**Comment by stevecambridge** [comment]
Is Bestial Curse a candidate? The failure and critical failure effects have different durations, which doesn't seem like it can be modified by rule elements.

**Comment by TikaelSol** [comment]
> Is Bestial Curse a candidate? The failure and critical failure effects have different durations, which doesn't seem like it can be modified by rule elements.

Not until we get duration modification in ItemAlteration

**Comment by simonward** [comment]
These need to be migrated and deleted:
Effect: Aeon Stone (Orange Prism) (Arcana)
Effect: Aeon Stone (Orange Prism) (Nature)
Effect: Aeon Stone (Orange Prism) (Occultism)
Effect: Aeon Stone (Orange Prism) (Religion)
Effect: Breach the Abyss - Acid
Effect: Breach the Abyss - Cold
Effect: Breach the Abyss - Electricity
Effect: Breach the Abyss - Fire
Effect: Breach the Abyss - Negative
Effect: Choose Weakness
Effect: Crocodile Form
Effect: Energy Ward
Effect: Snake Form
Spell Effect: Sweet Dream (Glamour)
Spell Effect: Sweet Dream (Insight)
Spell Effect: Sweet Dream (Voyaging)
Spell Effect: Weapon Surge (Major Striking)

**Comment by simonward** [comment]
```
Effect: Divine Weapon (Chaotic)
Effect: Divine Weapon (Evil)
Effect: Divine Weapon (Good)
Effect: Divine Weapon (Lawful)
```

**Comment by simonward** [comment]
```
Effect: Align Armament (Chaotic)
Effect: Align Armament (Evil)
Effect: Align Armament (Good)
Effect: Align Armament (Lawful)
```

**Comment by simonward** [comment]
```
Spell Effect: Elemental Form (Air)
Spell Effect: Elemental Form (Earth)
Spell Effect: Elemental Form (Fire)
Spell Effect: Elemental Form (Water)
```

**Comment by simonward** [comment]
```
Spell Effect: Primal Summons (Air)
Spell Effect: Primal Summons (Earth)
Spell Effect: Primal Summons (Fire)
Spell Effect: Primal Summons (Water)
```

**Comment by simonward** [comment]
```
Effect: Bespell Weapon
Effect: Bespell Weapon (Force)
Effect: Bespell Weapon (Mental)
Effect: Bespell Weapon (Void)
```

**Comment by simonward** [comment]
```
Spell Effect: Nature Incarnate (Green Man)
Spell Effect: Nature Incarnate (Kaiju)
```

**Comment by simonward** [comment]
```
Spell Effect: Shifting Form (Claws)
Spell Effect: Shifting Form (Climb Speed)
Spell Effect: Shifting Form (Darkvision)
Spell Effect: Shifting Form (Scent)
Spell Effect: Shifting Form (Speed)
Spell Effect: Shifting Form (Swim Speed)
```

**Comment by simonward** [comment]
`Effect: Cut from the Air`

**Comment by DocSchlock** [comment]
> Discern Secrets (Seek)
> Discern Secrets (Recall Knowledge)
> Discern Secrets (Sense Motive)

**Comment by simonward** [comment]
```
Effect: Rapid Evolution - Energy Gland
Effect: Rapid Evolution - Husk
```

**Comment by simonward** [comment]
`Spell Effect: Enlarge (Heightened 4th)`

**Comment by simonward** [comment]
`Effect: Discordant Voice`

**Comment by simonward** [comment]
```
Effect: Earplugs
Effect: Earplugs (PFS Guide)
```

**Comment by simonward** [comment]
```
Effect: Divine Infusion (Harm)
Effect: Divine Infusion (Heal)
```

**Comment by simonward** [comment]
```
Spell Effect: Swampcall (Critical Failure)
Spell Effect: Swampcall (Failure)
```

**Comment by simonward** [comment]
```
Spell Effect: Fungal Infestation (Failure)
Spell Effect: Fungal Infestation (Critical Failure)
```

**Comment by simonward** [comment]
`Effect: Divine Castigation`

**Comment by simonward** [comment]
```
Effect: Rune of Destruction (Greatsword)
Effect: Rune of Destruction (Longspear)
Effect: Rune of Destruction (Rock)
Effect: Rune of Flames (Greatsword)
Effect: Rune of Flames (Longspear)
Effect: Rune of Flames (Rock)
Effect: Rune of Smiting (Greatsword)
Effect: Rune of Smiting (Longspear)
Effect: Rune of Smiting (Rock)
```

**Comment by pedrogrullada** [comment]
```
Effect: Educate Allies (Self)
Effect: Demonic Blood Magic (Target)
Effect: Genie Blood Magic (Target)
Effect: Nymph Blood Magic (Target)
Effect: Shadow Blood Magic (Stealth)
Effect: Wyrmblessed Blood Magic (Status Penalty - Target)
```

**Comment by simonward** [comment]
```
Spell Effect: Animal Feature (Cat Eyes)
Spell Effect: Animal Feature (Claws)
Spell Effect: Animal Feature (Fish Tail)
Spell Effect: Animal Feature (Jaws)
Spell Effect: Animal Feature (Owl Eyes)
Spell Effect: Animal Feature (Wings)
```

**Comment by simonward** [comment]
```
Spell Effect: Tempest Form (Air)
Spell Effect: Tempest Form (Mist)
Spell Effect: Tempest Form (Water)
```

**Comment by simonward** [comment]
```
Effect: Emblazon Armament (Shield)
Effect: Emblazon Armament (Weapon)
Effect: Emblazon Energy (Weapon, Acid)
Effect: Emblazon Energy (Weapon, Cold)
Effect: Emblazon Energy (Weapon, Electricity)
Effect: Emblazon Energy (Weapon, Fire)
Effect: Emblazon Energy (Weapon, Sonic)
```

**Comment by pedrogrullada** [comment]
```
Effect: Bolera's Interrogation (Critical Failure)
```

**Comment by ammalagonc** [comment]
Effect: Current Spell (Air)
Effect: Current Spell (Water)

Become 

Effect: Current Spell

In #15275

**Comment by simonward** [comment]
```
Effect: Rowan Rifle (Cold)
Effect: Rowan Rifle (Electricity)
Effect: Rowan Rifle (Sonic)
```

**Comment by simonward** [comment]
```
Spell Effect: Elemental Motion (Air)
Spell Effect: Elemental Motion (Earth)
Spell Effect: Elemental Motion (Fire)
Spell Effect: Elemental Motion (Water)
```

**Comment by ammalagonc** [comment]
```
Draw Ire: Critical Failure
Draw Ire: Failure
```

https://github.com/foundryvtt/pf2e/pull/16135

**Comment by simonward** [comment]
```
Spell Effect: Traveler's Transit (Climb)
Spell Effect: Traveler's Transit (Fly)
Spell Effect: Traveler's Transit (Swim)
```

**Comment by iDantar** [comment]
```Effect: Change Shape (Werecreature)``` - Dedication already holds all REs, Effect is unreferenced
